src/js/app.js

```diff
@@ -91,6 +91,49 @@ $(document).ready(function() {
     }
   });
 
+  //Add toggle effect for sliding #menu
+  let showingMenu = false;
+  const $menu = $('#menu');
+  const $main = $('main');
+  const config2 = {
```

Comment:
I'm happy to see you're using `const` for things that don't change.
However you are no using `config2` anywhere?
And arguable `2` is not best name?